buns_store/build.gradle

```diff
@@ -23,11 +23,13 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.12.0'
```

Comment:
let's standardize the approach in dependencies:
There are 2 options available:
1. Using the long version of dependencies like `implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.12.0`
2. Or using the shortened version like: `implementation 'com.squareup.okhttp3:okhttp:4.12.0`

Both are fine but let's use them in all places in dependencies ;) (currently `spring` related deps have the shortened way and this one has the longer way)